### DISCUSSION TITLE
Enabling CORS for local font?

### DISCUSSION THREAD

**Discussion by Antonio-Laguna** [vercel/next.js#63174] [Help]
### Summary

I've realised that local fonts don't have CORS enabled. While this is normally fine, I have a usecase to serve the response from another place. Everything works fine but the local font.

The font is loaded as follows:

```ts
import localFont from 'next/font/local';

const zodiak = localFont( {
	variable: '--ff-serif',
	src: '../public/fonts/zodiak/Zodiak-Variable.ttf',
	weight: '400 900',
} );

export { zodiak };
```

I've tried to add this to `headers` on `next.config.js` but no luck:

```js
{
	source: '/_next/static/media/:path*',
	headers: [
		{ key: 'Access-Control-Allow-Origin', value: '*' },
		{
			key: 'Access-Control-Allow-Methods',
			value: 'GET,OPTIONS,PATCH,DELETE,POST,PUT',
		},
		{
			key: 'Access-Control-Allow-Headers',
			value: 'X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version',
		},
	],
},
```

Any help would be appreciated!

**Comment by smrubin** [comment]
@Antonio-Laguna I am having this same issue, did you come across a working solution by chance?

**Reply by Antonio-Laguna** [reply]
Sadly not, this lost priority for the client but it's a PITA
